August2020 updates (#13)
b2io/handbook: who-we-are.md

We typically work on smaller 2-6 month projects that use forward-leaning technologies. This allows our team to gain experience with the latest technologies in different industries. For projects that have longer durations we attempt to rotate team members so that the work does not feel stale, and so that more of the team has experience with any given project.

In addition we also provide access to online training opportunities and sponsor employee attendance at various technical conferences. We encourage community involvement through technical meetups and open source contributions.

> Career growth is facilitated and tracked. Individuals meet routinely with their supervisor to discuss career goals, track goal progress, and discuss feedback from the rest of the team. Our flat hierarchy means that no one has a direct manager. we do not review our teammates, We advocate for them.

Reviewer: Here and elsewhere I think we want to find a different word for "supervisor". It just doesn't feel like the right terminology for how Base Two is structured, but perhaps I'm being overly sensitive.

Reviewer 2: You are faster than I am, I was going to flag that.  I agree, but I couldn't figure out an alternate.  Any ideas?

> Career growth is facilitated and tracked. Individuals meet routinely with their supervisor to discuss career goals, track goal progress, and discuss feedback from the rest of the team. Our flat hierarchy means that no one has a direct manager. we do not review our teammates, We advocate for them.

Reviewer 3: It was Drew as part of the review process the last time, but conceivably it could be some other person at some point so just simplify to `team member`?
```suggestion
Career growth is facilitated and tracked. Individuals meet routinely with a team member to discuss career goals, track goal progress, and discuss feedback. Our flat hierarchy means that no one has a direct manager. We are accountable to, and advocate for each other.
```

Base Two is an internally transparent company. We make the majority of decisions regarding the company as a group and all employee's opinions are valued, sought after, and taken into consideration. All employees have access to the organization's financial information, including the [Base Two Salaries](https://b2io.slack.com/archives/C904HLD5L/p1516982712000070) document, which shows the team's compensation levels. We [meet once a week](./day-to-day.md#weekly-meeting) as a team to discuss a broad range of company-related topics.

#### Cost of Living Adjustments

Cost of living adjustments are applied each January based on the previous year's [Social Security COLA](https://www.ssa.gov/oact/cola/colaseries.html). The adjustment applies to all full-time employees as of 12/31 of the previous year.

#### Raises

If a team member believes that they (or any other member of the team) deserve a raise, then it should be brought up for discussion either in Buddy meetings, small groups, the Weekly Meeting, or whichever setting feels most comfortable. These discussions have tended to be informal in the past, and focus on the value and responsibilities that the team member provides.

> #### Conferences

Reviewer 2: Add the conferences you've attended while at B2 here! (I went to DrupalGC like a month before I started here, so I'm counting that.)
